net-misc/chrony/chrony-3.5-r3.ebuild

```diff
@@ -0,0 +1,150 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit systemd tmpfiles toolchain-funcs
+
+DESCRIPTION="NTP client and server programs"
+HOMEPAGE="https://chrony.tuxfamily.org/"
+SRC_URI="https://download.tuxfamily.org/${PN}/${P/_/-}.tar.gz"
+LICENSE="GPL-2"
+SLOT="0"
+
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ppc ~ppc64 ~sparc ~x86"
+IUSE="
+	+adns caps +cmdmon html ipv6 libedit +ntp +phc pps readline +refclock +rtc
```

Comment:
Don't you want to enable either USE=caps or USE=seccomp by default?

Reply:
Yes, I was being a bit cautious but it should be fine (especially caps).